qubesadmin/tools/qvm_start_daemon.py

```diff
@@ -699,6 +814,10 @@ def register_events(self, events):
                            self.on_connection_established)
         events.add_handler('domain-stopped', self.on_domain_stopped)
 
+        for event in ["property-set:audiovm", "property-pre-set:audiovm",
+                      "property-pre-del:audiovm"]:
```

Comment:
Why all three? especially, it should be only `property-set`, not `property-pre-set`, but then probably also `property-del` (or more intuitively `property-reset`), not `property-pre-del`.

Reply:
(based on direct chat) this is because changing audiovm changes also tags, which affects qrexec policy - after unsetting audiovm, old audiovm won't have access to VM's properties; and similarly, when setting new, it's accessible only after tags are updated. This could use a comment, or an explanation in the commit message.